site/content/posts/2021-11-09-E2E-Skeleton-Plugin.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Skip the Boilerplate: A Plugin Ready For Your Business Logic"
+image: /img/sonobuoy.svg
+excerpt: A new plugin helper library along with the new, upstream e2e-framework helps you jump straight into your testing your business logic. 
+author_name: John Schnake
+author_url: https://github.com/johnschnake
+categories: [kubernetes, sonobuoy, plugins]
+tags: ['Sonobuoy Team', 'John Schnake']
+date: 2021-11-09
+slug: plugin-starter
+---
+
+## Introduction
+
+When you decide to test your Kubernetes native code it can be a big chore. When I've been in that position, I do one of the following:
```

Comment:
`".. When I've been in that position.."` -> What position ? (maybe `When I have to write tests`)